gcloud/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Google LLC
```

Comment:
Would it be better to put this inside the `release` directory? It seems strange to create a new top-level `gcloud` directory.